test/github-issues/5407/entity/EnumUser.ts

```diff
@@ -0,0 +1,17 @@
+import {Column} from "../../../../src";
+import {Entity} from "../../../../src/decorator/entity/Entity";
+import { User } from "./User";
+
+enum Gender {
+  UNSPECIFIED = "",
+  FEMALE = "F",
+  MALE = "M",
+  OTHER = "O",
+}
+
+@Entity()
+export class EnumUser extends User {
```

Comment:
Is this needed for the test?  If not, can we remove it?

Reply:
Did you mean testing enum? I just checked it is covered in `test\functional\database-schema\enums\enums.ts`. It can be removed.